mlflow/utils/env.py

```diff
@@ -8,3 +8,24 @@ def get_env(variable_name):
 def unset_variable(variable_name):
     if variable_name in os.environ:
         del os.environ[variable_name]
+
+
+class EnvironmentVariable:
```

Comment:
I think we can move this in `mlflow/environment_variables.py`.

Reply:
```suggestion
class _EnvironmentVariable:
```

Users don't need to use this class.